hosted/aks/k8s_chart_support/k8s_chart_support_import_test.go

```diff
@@ -17,18 +15,9 @@ var _ = Describe("K8sChartSupportImport", func() {
 	var cluster *management.Cluster
 
 	BeforeEach(func() {
-		var err error
-		err = helper.CreateAKSClusterOnAzure(location, clusterName, k8sVersion, "1")
+		err := helper.CreateAKSClusterOnAzure(location, clusterName, k8sVersion, "1", helpers.GetCommonMetadataLabels())
 		Expect(err).To(BeNil())
-
-		aksConfig := new(helper.ImportClusterConfig)
-		config.LoadAndUpdateConfig(aks.AKSClusterConfigConfigurationFileKey, aksConfig, func() {
-			aksConfig.ResourceGroup = clusterName
-			aksConfig.ResourceLocation = location
-			aksConfig.Tags = helper.GetTags()
-		})
-
-		cluster, err = helper.ImportAKSHostedCluster(ctx.RancherClient, clusterName, ctx.CloudCred.ID, false, false, false, false, map[string]string{})
+		cluster, err = helper.ImportAKSHostedCluster(ctx.RancherClient, clusterName, ctx.CloudCred.ID, location, helpers.GetCommonMetadataLabels())
```

Comment:
```suggestion
		cluster, err = helper.ImportAKSHostedCluster(ctx.RancherClient, clusterName, ctx.CloudCred.ID, location, helpers.GetCommonMetadataLabels())
```
```suggestion
		cluster, err = helper.ImportAKSHostedCluster(ctx.RancherClient, clusterName, ctx.CloudCred.ID, location, helpers.GetCommonMetadataTags())
```
Not to be confused with Cluster-level labels

Reply:
Label for one provider is a tag for another, AKS and EKS prefer calling it `tag`, GKE uses `labels`. I can't seem to think of any generic name that suits all 3.